File: java/home_work_1/task7.java
import java.util.*;
class Task1 {
    public static void main(String[] args) {
        
            int a,b;
            System.out.println("Please Enter a");
            Scanner sa = new Scanner(System.in);
            System.out.println("Please Enter b");
            Scanner sb = new Scanner(System.in);
            a = sa.nextInt();
            b = sb.nextInt();


	a = a + b;
	b = a - b;
	a = a - b;
	System.out.println("changed a and b are a = " + a + " b = " + b);
    }
}

File: cpp/home_work_1/task7.cpp
#include <iostream>

int main() {
    int a,b;

    std::cout << "Please enter a ";
    std::cin >> a;
    std::cout << "Please enter b ";
    std::cin >> b;
    std::cout << std::endl;

    a = a+b;
    b = a-b;
    a = a-b;

    std::cout << "changed a and b numbers are " << a << " <->  " << b << std::endl;

    return 0;
}
